persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/client/ClientDAO.kt

```diff
@@ -83,7 +83,7 @@ interface ClientDAO {
      * @param userIds the set of users
      * @return a map of users and their clients
      */
-    suspend fun recipientsIfTheyArePartOfConversation(
+    suspend fun recipientsIfTHeyArePartOfConversation(
```

Comment:
```suggestion
    suspend fun recipientsIfTheyArePartOfConversation(
```

Typo? Seems to have affected multiple stuff